app/tray/wintray/winclass.go

```diff
@@ -0,0 +1,45 @@
+//go:build windows
```

Comment:
Amazing this was all doable in Go

Reply:
Yup.  This builds on prior art from https://github.com/getlantern/systray and https://github.com/hallazzang/go-tray-icons-tutorial